@Test public void error(){
  List<Throwable> errors=TestHelper.trackPluginErrors();
  try {
    Flowable.<Integer>error(new TestException()).parallel().reduce(new BiFunction<Integer,Integer,Integer>(){
      @Override public Integer apply(      Integer a,      Integer b) throws Exception {
        return a + b;
      }
    }
).test().assertFailure(TestException.class);
    assertTrue(errors.isEmpty());
  }
  finally {
    RxJavaPlugins.reset();
  }
}
